metabase/metabase
# More UI updates

Closes https://github.com/metabase/metabase/issues/45427

### Description
1. Update dark [theme text colors](https://www.notion.so/metabase/Re-design-transparent-theme-for-static-embedding-and-public-links-a28e98a6e5c64301a48387c7b333d569?pvs=4#506e8839f4b746c5b9d92d84843c379b). See [the discussion](https://metaboat.slack.com/archives/C02H619CJ8K/p1720698749676689?thread_ts=1720695025.660599&cid=C02H619CJ8K).
2. Update powered by [Metabase design.](https://www.figma.com/design/OamRqg07JOoC4bUqfCkh4t/SDK-Reference-APP?node-id=2454-3169&t=cMTEF4IOFRnptliJ-0) See the [discussion.](https://metaboat.slack.com/archives/C02H619CJ8K/p1720774089781489?thread_ts=1720092452.132689&cid=C02H619CJ8K)

### How to verify

1. Run storybook
2. Go to http://localhost:6006/?path=/story/embed-publicorembeddeddashboardview--light-theme-default
3. View other stories as well.
4. Go to any public actions, and you should see the light version of the badge there.

You should see that the text contrast is better and Metabase badge is already updated.

### Demo
![image](https://github.com/user-attachments/assets/e5062f38-9de8-4352-bded-fe61fdc0e5d3)
![image](https://github.com/user-attachments/assets/848ce599-cf27-4218-bd07-bd8f05656c9c)
![image](https://github.com/user-attachments/assets/86e020b7-c179-4835-b1b4-95778a8797ab)
![image](https://github.com/user-attachments/assets/2c6bf6bf-bf78-4534-b00d-f7247970bf88)


### Checklist

- [x] Tests have been added/updated to cover changes in this PR (Loki snapshots)
